ClinGen/clincoded
# Connects to #1801. Connects to #1807. Connects to #1818. GDM and Interpretation stats report.

**Technical notes:**
1. Production data is required for testing #1818.
2. Because the **admin** feature is still in its early development, the side nav is not yet functional.
3. Included in this PR are new users being added for the R23 release, as well as new affiliations added to existing users.

**Steps to test #1801:**
1. Sign in and assign yourself to affiliation **10026**.
2. Sign out and sign in again. Expect to see the affiliation name to be changed from **ClinGen Biocuration Core** to **General Gene Curation EP** in the affiliation modal selection dropdown.

**Steps to test #1807:**
1. Assign yourself to affiliations **10031** and **10032**.
2. Sign out and sign back in again. Expect to see the affiliation names **Congenital Myopathies EP** and **Stanford Center for Inherited Cardiovascular Disease** in the affiliation modal selection dropdown.

**Steps to test #1818:**
1. Sign in as admin and go to ^/admin.
2. On the landing page, expect to see a **Reports** selection dropdown.
3. Select either the **Interpretation Stats for Expert Panels** or the **Gene-Disease Record Stats for Expert Panels** option and click the **Submit** button.
4. Expect to see a data table being rendered given the number of currently active affiliations.
5. Expect to be able to sort the columns by clicking on the table header cells.
6. Expect to be able to download a `.csv` file of the data by clicking on the blue **Export data to .csv** button at the bottom of the table.
